portfolio/portfolio.go

```diff
@@ -37,8 +37,9 @@ func GetEthereumBalance(address string) (EthplorerResponse, error) {
 	urlPath := fmt.Sprintf(
 		"%s/%s/%s?apiKey=freekey", ethplorerAPIURL, ethplorerAddressInfo, address,
 	)
+
 	result := EthplorerResponse{}
-	return result, common.SendHTTPGetRequest(urlPath, true, false, &result)
+	return result, common.SendHTTPGetRequest(urlPath, true, true, &result)
```

Comment:
Can make an exported Verbose var for this so we can set it up upon portfolio manager initialisation